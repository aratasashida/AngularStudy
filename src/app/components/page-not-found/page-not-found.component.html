<h1>
    Page Not Found!!
    スナップショットをfailueさせてみる
</h1>
