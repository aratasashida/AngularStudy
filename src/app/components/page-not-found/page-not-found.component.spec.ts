import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { PageNotFoundComponent } from './page-not-found.component';

describe('PageNotFoundComponent', () => {
  let component: PageNotFoundComponent;
  let fixture: ComponentFixture<PageNotFoundComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [ PageNotFoundComponent ]
    })
    .compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(PageNotFoundComponent);
    component = fixture.componentInstance;
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });

  it('snapshot test', () => {
    fixture.detectChanges();
    const nativeElement = fixture.debugElement.nativeElement;
    expect(nativeElement).toMatchSnapshot();
  });
});
