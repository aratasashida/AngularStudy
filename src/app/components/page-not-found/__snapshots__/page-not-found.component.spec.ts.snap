// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFoundComponent snapshot test 1`] = `
<div
  id="root1"
>
  <h1>
     Page Not Found!!

  </h1>
</div>
`;
